apps/configurator/README.md

```diff
@@ -35,6 +35,37 @@ You can:
 - Assign skills (see skills config) to each character: basic, ultimate, dash.
 - Tune character params: speed, health, stamina, etc.
 
+Its mandatory to activate a character to have a basic skin assigned, for example, if you want to add Uren you should add this this lines of code:
+
+- Set `active` to `true` in `seeds.exs`
+charactername_params = %{
+  name: "valtimer",
+  active: true,
+  ...
+  ...
+}
+
+- 4.2 Create a default skin (if needed)
+charactername_basic_params = %{
+  is_default: true,
+  name: "Basic",
+  character_id: Enum.find(characters, fn c -> c.name == "charactername" end).id
+}
+- Add `charactername_basic_params` to the skins list
+Find the **Insert skins** section and add the new character's basic skin:
+
+Insert skins
+[
+  h4ck_fenix_params,
+  h4ck_basic_params,
+  valtimer_frostimer_params,
+  valtimer_basic_params,
+  kenzu_black_lotus_params,
+  kenzu_basic_params,
+  otix_corrupt_underground,
+  otix_basic_params
+]
```

Comment:
Maybe we can replace all this by just saying that if you create a new character, you have to assign a default skin to it or it won't work, then show how to do it (like you do here).
Also, it would be nice to mention that this should be in the Configurator (again, these are Configurator's app docs), because now you can't create skins from it. If you add that, create an issue and link it here as well.